addons/hr_holidays/models/hr_leave.py

```diff
@@ -1383,6 +1383,19 @@ def action_documents(self):
             'domain': domain
         }
 
+    def action_new_time_off(self):
```

Comment:
Is it normal to have new code here ?

Reply:
this seems to be a rebase error

Reply:
@mart-e Can you check this ?

Reply:
@tde-banana-odoo it's to replace the only call to `get_view_id` in Odoo